app/ledger.js

```diff
@@ -859,8 +859,8 @@ var enable = (paymentsEnabled) => {
     // change undefined include publishers to include publishers
     appActions.enableUndefinedPublishers(synopsis.publishers)
 
-    fs.unlink(pathName(publisherPath), (err) => { if ((err) && (err.code !== 'ENOENT')) console.log(err) })
-    fs.unlink(pathName(scoresPath), (err) => { if ((err) && (err.code !== 'ENOENT')) console.log(err) })
+    fs.unlink(pathName(publisherPath), (err) => { if ((err) && (err.code !== 'ENOENT')) console.error(err) })
+    fs.unlink(pathName(scoresPath), (err) => { if ((err) && (err.code !== 'ENOENT')) console.error(err) })
```

Comment:
similar to above, would be nice to log that error happened deleting a file (and sharing which file failed)